### DISCUSSION TITLE
CFS Library

### DISCUSSION THREAD

**Discussion by Tagliente-Simone** [contiki-ng/contiki-ng#1748] [Q&A] [answered]
Hi everyone! I am using the cfs library to read a file containing numbers. I use the function cfs_read to read the file but I realized that that function reads all the file and put it in a buffer string. Is there a way that allows me to read the file line by line so that I can process one line at time?

**Comment by nvt** [comment, marked as answer]
No, the line processing has to be done manually because there is no line-based file reading functionality in CFS.

**Reply by Tagliente-Simone** [reply]
Okay thank you
